common/meta/raft-store/src/state_machine/sm.rs

```diff
@@ -848,10 +848,10 @@ impl StateMachine {
         let seq_dbi = txn_db_lookup
             .get(&(DatabaseLookupKey::new(tenant.to_string(), db_name.to_string())))?
             .ok_or_else(|| {
-                MetaStorageError::AppError(AppError::UnknownDatabase(UnknownDatabase::new(
+                AppError::UnknownDatabase(UnknownDatabase::new(
                     db_name.to_string(),
                     "txn_get_database_id".to_string(),
-                )))
+                ))
             })?;
```

Comment:
`From<AppError> for MetaStorageError` is already implemented. 
Thus IMHO this change has nothing to do with this issue. not?